scavenger-agent-java/src/integrationTest/java/integrationTest/util/AgentLogAssertionUtil.java

```diff
@@ -0,0 +1,31 @@
+package integrationTest.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.matchesPattern;
+
+import java.util.regex.Pattern;
+
+public class AgentLogAssertionUtil {
+    public static void assertSampleAppOutput(String stdout) {
+        assertThat(stdout, matchesPattern(logPattern("sample.app.SampleApp", "SampleApp starts")));
+        assertThat(stdout, matchesPattern(logPattern("sample.app.SampleApp", "2+2=4")));
+        assertThat(stdout,
+            matchesPattern(logPattern("sample.app.SampleAspect", "Before execution(void sample.app.SampleService1.doSomething(int))")));
+        assertThat(stdout, matchesPattern(logPattern("sample.app.SampleService1", "Doing something 1")));
+        assertThat(stdout,
+            matchesPattern(logPattern("sample.app.SampleAspect", "Before execution(void sample.app.SampleService2.doSomething(int))")));
+        assertThat(stdout, matchesPattern(logPattern("sample.app.SampleService2", "Doing something 2")));
+        assertThat(stdout, matchesPattern(logPattern("sample.app.NotServiceClass", "Doing something 4")));
+        assertThat(stdout, matchesPattern(logPattern("sample.app.SampleAspect", "After execution(void sample.app.SampleService2.doSomething(int))")));
+        assertThat(stdout, matchesPattern(logPattern("sample.app.SampleAspect", "After execution(void sample.app.SampleService1.doSomething(int))")));
+        assertThat(stdout, matchesPattern(logPattern("sample.app.SampleApp", "Exit")));
+    }
+
+    public static void assertDisabled(String stdout) {
+        assertThat(stdout, matchesPattern(logPattern("com.navercorp.scavenger.javaagent.ScavengerAgent", "[scavenger] scavenger is disabled")));
+    }
+
```

Comment:
assertDisabled  is only used for ConfigTest
I believe hiding asserting logic in different files is not good idea.
It makes the validation not explicit.

Reply:
Moved the method to relating class `ConfigTest` in [23f7752](https://github.com/naver/scavenger/pull/71/commits/23f7752f4d86902ae2579a40bfd691c3cad3b8ee).